providers/tutorial-create-a-provider.md

```diff
@@ -0,0 +1,600 @@
+---
+title: (Tutorial) Create a provider
+category: 63d7e8bdbf7b4b0e0745e823
+slug: tutorial-create-a-provider
+---
+
+# Create a provider
+
+This tutorial aims to help you create and modify your first provider. It will
+not touch "advanced" cell types like the graph and the log cells, but will help
+you integrate a third-party resource so that any output from it can be added to
+a notebook with only a `/` command (we call those "slash commands").
+
+This tutorial focuses on writing the provider in Rust, to be able to use the
+convenience macros from Fiberplane Plugin Development Kit (PDK). Writing
+providers in different languages will be addressed in a different guide.
```

Comment:
as we currently only support Rust for provider development (as I understand it) - we should probably just say "Provider Development Kits for other languages are coming soon"

Reply:
Fair enough, I might even choose "later" rather than "soon" since we don't have any ETA on PDK for other languages (it mostly depends on what bindings `fp-bindgen` can generate, so TS is probably close to ready if we tackle it; but all others are far away)